aws/rust-runtime/aws-config/src/credential_process.rs

```diff
@@ -3,57 +3,20 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#![cfg(feature = "credentials-process")]
```

Comment:
question because I don't know: Is this strictly equivalent to adding the same flag to the `credential_process` `mod` declaration?

Reply:
yeah